css/amp.css

```diff
@@ -167,6 +167,24 @@ html.i-amphtml-singledoc.i-amphtml-ios-embed-sd > body {
   display: inline-block;
 }
 
+.i-amphtml-blur-loading .i-amphtml-fill-content {
+  opacity: 0 !important;
+}
+
+.i-amphtml-blur-loaded .i-amphtml-fill-content {
+  opacity: 1 !important;
+  transition: opacity 1s cubic-bezier(0.33, 0.33, 1, 1) !important;
+}
+
+.i-amphtml-blur-loading .i-amphtml-blur {
+  opacity: 1 !important;
+}
+
+.i-amphtml-blur-loaded .i-amphtml-blur {
+  opacity: 0 !important;
+  transition: opacity 1s cubic-bezier(0.33, 0.33, 1, 1) 1s !important;
+}
```

Comment:
Do we need to animate both the placeholder and the image instead of just the latter?

Reply:
I know I talked to you in person about this, but the reason I think it used the animation on both before was because since the blur is a child, it was rendered on top of the parent so the blur would appear even on top of the loaded image. So the animation I could see was actually the fade out of the blur. I did adjust this and was able to remove the placeholder animation by setting the z-index of .i-amphtml-blur to -1. Is this an ok solution or could introducing the z-index value cause other problems?